packages/trpc/server/routers/viewer/bookings/confirm.handler.ts

```diff
@@ -214,7 +214,7 @@ export const confirmHandler = async ({ ctx, input }: ConfirmOptions) => {
       : user.destinationCalendar
       ? [user.destinationCalendar]
       : [],
-    requiresConfirmation: booking?.eventType?.requiresConfirmation ?? false,
+    requiresConfirmation: !confirmed,
```

Comment:
What is the reason for this change?

Reply:
This is also true for the email when the event gets confirmed.